inappmessaging/src/test/java/com/rakuten/tech/mobile/analytics/Event.kt

```diff
@@ -0,0 +1,15 @@
+package com.rakuten.tech.mobile.analytics
+
+/**
+ * An event to be tracked.
+ * @param name Event's name
+ * @param parameters Event parameters
+ */
+class Event(name: String, parameters: Map<String, Any>) {
```

Comment:
How is this class related to `Event` from `src/main` ? (That might be a newbie question)

Reply:
Hi @imaginaris,
Since the Event class from src/main is not recognized from the test resource file at compile-time, I added this fake Event class to generate it.

Reply:
Ok, got it. It doesn't look like a fake :D